src/main/java/pixelperfect/spaceship/Route.java

```diff
@@ -0,0 +1,50 @@
+package pixelperfect.spaceship;
+
+import java.util.ArrayList;
+
+/**
+ * A route through space, to be traversed by the spaceship.
+ * 
+ * @author Jesse Tilro
+ *
+ */
+public class Route {
+
+  private long timestamp;
+  private long duration;
+  private ArrayList<RouteNode> nodes;
+
+  /**
+   * Construct a new Route instance with a specified duration in milliseconds.
+   * 
+   * @param duration
+   *          The total duration of the route.
+   */
+  public Route(long duration) {
+    this.timestamp = System.currentTimeMillis();
+    this.duration = duration;
+    this.nodes = new ArrayList<RouteNode>();
+  }
+
+  /**
+   * Determine whether the route is completed.
+   * 
+   * @param now
+   *          Current timestamp in milliseconds.
+   * @return Boolean.
+   */
+  public boolean isCompleted(long now) {
+    return (now >= timestamp + duration);
+  }
+
+  /**
+   * Randomly generate a Route (which is an aggregation of RouteNodes) using the random
+   * RouteGenerator.
+   * 
+   * @return A randomly generated route.
+   */
+  public static Route generateRoute() {
+    return new Route(42);
```

Comment:
Also, we can just give a RouteGenerator object as private atribute of the class and initiate it when a Route is created. 
Then I would like to add the fact that this class should be a Singleton: only one instance of the class can be/should be alive at the time.